### PR TITLE
WAS-585 Add partner page

### PR DESCRIPTION
**Pull request by rickclare**
This complements PR https://github.com/EnvironmentAgency/waste-exemptions/pull/144

### CODE REVIEW

**Comment by rickclare** [comment]
@Cruikshanks over to you for PR review (this complements PR https://github.com/EnvironmentAgency/waste-exemptions/pull/144)

File: Gemfile [hunk 1]
```diff
@@ -10,7 +10,10 @@ gemspec
 # Once the development stable we should move this to the gemspec
 gem 'digital_services_core',
     git: 'https://github.com/EnvironmentAgency/digital-services-core',
-    branch: 'develop'
+    branch: 'develop',
+    ref: 'd98324f' # Temporarily restricting to a commit hasn't broken DigitalServicesCore::Contact#organisation?
```
Comment (Cruikshanks) [inline Gemfile]: Don't understand the need to reference a particular commit, the comment isn't clear to me. Could you elaborate?

File: app/validators/waste_exemptions_shared/grid_reference_validator.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@ def initialize(location)
     cattr_accessor :site_info_max_length do 500 end
     cattr_accessor :site_info_min_length do 3 end
 
-    GRID_REFERENCE_REGEX = /\A[a-zA-Z]{2}\d{6,10}\z/i.freeze
+    GRID_REFERENCE_REGEX = /\A[a-zA-Z]{2}\d{6,10}\z/i
```
Comment (Cruikshanks) [inline app/validators/waste_exemptions_shared/grid_reference_validator.rb]: A number of recent commits have added `.freeze` to constants. I'm curious why this one has been removed.

File: circle.yml [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+    - git config --global user.name "Test Example" # overcommit Author
+    - npm install -g jshint coffeelint # overcommit packages
+
 database:
   override:
     - cp config/database.circle-ci.yml config/database.yml
-    - bundle exec rake db:create:all --trace
-    - bundle exec rake db:migrate --trace
+    - RAILS_ENV=test bundle exec rake db:create:all --trace
+    - RAILS_ENV=test bundle exec rake db:migrate --trace
```
Comment (Cruikshanks) [inline circle.yml]: Is it worth adding a call to `rake db:seed` here? Appreciate we haven't really got anything now but when the seeding PR's get in would this be a way of testing them as part of CI?

**Comment by Cruikshanks** [comment]
As usual just a few random questions and then its ready to go @rickclare

File: Gemfile (hunk 1 above)
Comment (rickclare) [inline Gemfile]: I had to add the ref, as specs were falling. This is due to Timvl making changes to Digital_services_core and Waste_exemptions_shared, but only Digital_services_core changes have been merged into develop as yet.

Hurry up with your pull request request reviews and then I can remove this ref 😄

File: circle.yml (hunk 3 above)
Comment (rickclare) [inline circle.yml]: I would say no - Seeds shouldn't be added to a test database, add they could inadvertently affect specs that rely on a clean database.

File: app/validators/waste_exemptions_shared/grid_reference_validator.rb (hunk 2 above)
Comment (rickclare) [inline app/validators/waste_exemptions_shared/grid_reference_validator.rb]: Robocop had started composing that "Freeze" has no effect on regular expressions (even for a constant). Hence I had to remove this freeze to pass overcommit

**Comment by rickclare** [comment]
@Cruikshanks hopefully I've addressed your questions ok?

**Pull request merged**